### PR TITLE
Bug 1301401 - updating to chainOfTrust version 1 format

### PR DESCRIPTION
**Pull request by petemoore**
Hey @gregarndt I figured the easiest way to test these changes is to open a PR and let taskcluster run the test suite(s). I haven't tested locally, so it might take a few commits to get it right. I'll ping you when it is ready for final review!

CC: @escapewindow @walac

### CODE REVIEW

File: lib/features.js [hunk 1]
```diff
@@ -16,13 +16,14 @@ const features = {
     module: require('./features/local_live_log')
   },
 
-  generateCertificate: {
-    title: 'Enable generation of a certificate for Chain of Trust',
-    description: 'A certificate should be generated which will include information ' +
-                 'for downstream tasks to build a level of trust for the artifacts ' +
-                 'produced by the task and the environment it ran in.',
+  chainOfTrust: {
+    title: 'Enable generation of a openpgp signed Chain of Trust artifact',
```
Comment (gregarndt) [inline lib/features.js]: This is missing the concatenation and will break trying to run the worker.

File: lib/upload_to_s3.js [hunk 2]
```diff
@@ -120,5 +120,5 @@ export default async function uploadToS3 (
     tmp.unlink();
   }
 
-  return `sha256:${digest}`;
+  return `${digest}`;
```
Comment (gregarndt) [inline lib/upload_to_s3.js]: just return digest then, no need to use a template string

File: lib/features.js (hunk 1 above)
Comment (petemoore) [inline lib/features.js]: Nice spot! Thanks.

File: lib/upload_to_s3.js (hunk 2 above)
Comment (petemoore) [inline lib/upload_to_s3.js]: Ah yes! Thanks.

**Comment by petemoore** [comment]
@gregarndt Tests have finished, we have 38 failures - are they genuine problems related to this PR? Thanks!

https://public-artifacts.taskcluster.net/V6iUaCaUQjOpRThfyQnnHQ/0/public/logs/live_backing.log

**Comment by gregarndt** [comment]
I closed/opened the PR to retrigger another graph creation.  Retriggering an individual task doesn't work because right now we're using encrypted environment variables that encode the task ID into them.

**Comment by petemoore** [comment]
@gregarndt can you check https://tools.taskcluster.net/task-inspector/#mMF4b_SSTFCHzyz6jViRlg/0 and https://tools.taskcluster.net/task-inspector/#Tpt4eVdLTvWL3b1nqXg1IQ/0 to see if you think these are related. I think these might also be timeout/socket issues.

**Comment by gregarndt** [comment]
So it looks like there are 2 unique errors, one of them wasn't caused by what you did from what I can see, and the other one I fixed in another PR.  I think you're good here.

**Comment by petemoore** [comment]
@gregarndt Great! Does that mean we can merge?

**Comment by gregarndt** [comment]
Ok, merged, but won't be deployed immediately.  It's Friday so I would prefer to hold off to land changes on Monday.

**Pull request merged**